projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js

```diff
@@ -28,6 +28,8 @@ import origamiPlane from './icons/origami-plane';
 import { isUserConnected } from './lib/connection';
 import UpgradePrompt from './upgrade-prompt';
 
+const isInBlockEditor = window?.Jetpack_Editor_Initial_State?.screenBase === 'post';
```

Comment:
I don't know the API, so wondering how reliable this condition is. I was taking a look at the [is_block_editor()](https://developer.wordpress.org/reference/classes/wp_screen/is_block_editor/) method. Just FYI.

Reply:
Is block editor returns `true` in the site editor.